radio/src/datastructs.h

```diff
@@ -651,6 +651,20 @@ PACK(struct ModelData {
 
   char modelRegistrationID[PXX2_LEN_REGISTRATION_ID];
 
+//OW
+#if defined(TELEMETRY_MAVLINK)
+  uint16_t _mavlinkEnabled:1; // currently not used
+  uint16_t mavlinkRssi:1;
+  uint16_t _mavlinkDummy:2; // currently not used
+  uint16_t mavlinkMimicSensors:3; // currently just off/on, but allow e.g. FrSky, CF, FrSky passthrough.
+  uint16_t mavlinkRcOverride:1;
+  uint16_t _mavlinkGpsIcon:1; // currently not used
+  uint8_t  mavlinkRssiScale;
+  uint8_t  _mavlinkDummy2; // currently not used
```

Comment:
The items which are `currently not used` shall be removed. Those needed for padding use the `Spare` suffix. Would it be thinkable to pack the `mavlinkRssiScale` into only 4 bits, so we could get rid of the spare bits and save another 8bits?